components/CategoryItem.js

```diff
@@ -97,7 +102,7 @@ const CategoryTitle = styled.h2`
 `
 
 const CategoryItem = (props) => {
-  const { title, srcImage = '', categoryColor = '#428CD4', isLoading, className } = props
+  const { title, srcImage = '', categoryColor = '#428CD4', isLoading, className, id = '1' } = props
```

Comment:
We should not add a default id, so that the App redirect the user to the Not Found Page, or what do you think?